openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcesses.scala

```diff
@@ -708,14 +719,13 @@ class OpenEOProcesses extends Serializable {
         SpacePartitioner[K](kb)
       }
     } else {
+      // At least one partitioner is undefined.
       logger.info(s"Merging cubes with partitioners: ${leftCube.partitioner} - ${rightCube.partitioner} - many band case detected: $manyBands")
-      if(manyBands) {
```

Comment:
why the immediate removal of manyBands?
I would prefer a more incremental change that takes things one step at a time, allowing to release this one faster without too much worries.

Reply:
ManyBands returned either ByTileSpatialPartitioner or SpaceTimeByMonthPartitioner which are both special cases of the configurable partitioners so I thought it was safe. But of course the new partitioner has to be more thoroughly tested to ensure it doesn't actually exceed its maximum allowed memory size so I will revert the manyBands case.

Reply:
Pushed a fix.